packages/framework/src/Component/Elasticsearch/IndexFacade.php

```diff
@@ -283,4 +242,71 @@ public function exportIds(AbstractIndex $index, IndexDefinition $indexDefinition
 
         $this->sqlLoggerFacade->reenableLogging();
     }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Component\Elasticsearch\IndexDefinition $indexDefinition
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     */
+    protected function createForIndexWhenNeeded(IndexDefinition $indexDefinition, OutputInterface $output): void
```

Comment:
I think better naming would be `createForIndexIfNotExists`

"When needed" feels ambiguous

Reply:
Disagree. The method is creating a new index when there is no existing alias. Suggested method name is misleading.

Reply:
renamed to `createIndexWhenNoAliasFound`